### PR TITLE
Fix seeds with avatar and time_zone to fix validation failures

### PR DESCRIPTION
**Pull request by jespr**
The seeds took a nose dive after validations for time_zone and avatar were added. This fixes that. Thoughts on avatar files for the `seeds.rb` file to be placed in `test/files`

### Pull request checklist

- [ ] Your code contains tests relevant for code you modified
- [x] You have linted and tested the project with `bin/check`

### CODE REVIEW

**Comment by jespr** [comment]
Let me add a few more things to the seed before we merge :)

File: db/seeds.rb [hunk 1]
```diff
@@ -7,49 +7,73 @@ def create_user!(name)
   )
 end
 
+def create_developer!(attributes, avatar_name:)
```
Comment (joemasilotti) [inline db/seeds.rb:10]: If `avatar_name:` is required we can pass it with `attributes` so we don't have to use two hashes when calling this method.

```
def create_developer!(attributes)
  avatar_name = attributes.delete(:avatar_name)
  Developer.create!(attributes) do |developer|
    developer.avatar.attach(io: File.open(Rails.root.join("test/files/#{avatar_name}")), filename: avatar_name)
  end
end
```

File: db/seeds.rb [hunk 2]
```diff
@@ -7,49 +7,73 @@ def create_user!(name)
...
+  developer = Developer.new(attributes)
+  developer.avatar.attach(io: File.open(Rails.root.join("test/files/#{avatar_name}")), filename: avatar_name)
```
Comment (joemasilotti) [inline db/seeds.rb:12]: We already have some images in `test/fixtures/files/` – I'd rather use those instead of adding new ones.

**Review by joemasilotti — CHANGES_REQUESTED** [review]
Thanks for the PR! I didn't even realize the seeds were broken.

File: db/seeds.rb (hunk 2 above)
Comment (jespr) [reply db/seeds.rb:12]: So just `lovelace.jpg` for all the devs, and basecamp for the company? or?

Comment (jespr) [reply db/seeds.rb]: Added the changes :)

**Review by joemasilotti — CHANGES_REQUESTED** [review]
I'd like to still use the images you uploaded – can you move them to `test/support/fixtures/`? Sorry if that was confusing in my first review.

Also, I think we can extract another helper for attaching an image.

```ruby
def attach(filename, to:)
  to.attach(io: File.open(Rails.root.join("test/fixtures/files", filename), filename: filename)
end

attach("basecamp.png", to: business.avatar)
```

**Comment by jespr** [comment]
Doh! I completely misunderstood! This makes more sense! Let me add that

**Comment by jespr** [comment]
@joemasilotti I think that's it

**Comment by joemasilotti** [comment]
I made one small commit to optimize the images (why not?) and remove duplicates.

Thanks again for the contribution! I'll merge this in as soon as CI passes.

**Pull request merged**